### PR TITLE
Bilby information and updated samplers

### PR DESCRIPTION
**Pull request by franciswebb1**
All four samplers within bilby now working. Have updated site info removing old emcee and replacing with bilby information.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -180,8 +180,10 @@
...
+    <h3>Do you have some data and a model that you want to fit? Well here's the website for you (see <a href="#caveats">caveats</a>)!
+On this website you can input a model function defined by a set of parameters, including those that you want fit, as well as your data, and it will run a statisical sampling algorithm to estimate the posterior probability distributions of those parameters.<br><br>
+This site makes use of the Bayesian inference python package <a href="https://lscsoft.docs.ligo.org/bilby/index.html">Bilby</a> to access statisical samplers.
+Beyond <a href="https://en.wikipedia.org/wiki/Markov_chain_Monte_Carlo">Markov chain Monte Carlo</a> (MCMC), users are able to select from a variety of statistical <a href="#id_sampler_input">samplers</a> and it is encouraged to trial a variety to achieve the best fit for your model.</h3>
```
Comment (mattpitkin) [inline index.php]: I'd slightly change the working "... to achieve the best fit for your model...", and it sounds a bit like one sampler might actually give a "better fit", whereas they _should_ all achieve the same fit. So, maybe change "best fit" to "best performance".

File: index.php [hunk 2]
```diff
@@ -461,7 +464,11 @@
   </p>
   <br>
 
-  <h3 class="text-left" id="id_mcmc_header">Sampler Inputs</h2>
+  <h3 class="text-left" id="id_sampler_input">Sampler Inputs</h2>
+  Through <a href="https://lscsoft.docs.ligo.org/bilby/index.html">Bilby</a> one can select from a variety of statistical samplers, each utilising a slightly different algorithm to model the parameters. The samplers available are broken down into
```
Comment (mattpitkin) [inline index.php]: Change "to model the parameters" to "to sample from the parameter space"

File: theonlinemcmc/__init__.py [hunk 3]
```diff
@@ -4,4 +4,4 @@
 from .postprocessing import *
 from .errorcodes import *
 from .errorpage import *
-from .database import *
+from .database import *
```
Comment (mattpitkin) [inline theonlinemcmc/__init__.py]: Add a blank new line at the end of the file.

File: theonlinemcmc/postprocessing.py [hunk 4]
```diff
@@ -296,4 +315,4 @@ def postprocessing(postsamples, variables, abscissa, abscissaname, data, email,
   fp.close()
   
   # email the page
-  emailresponse(email, outdir)
+  emailresponse(email, outdir)
```
Comment (mattpitkin) [inline theonlinemcmc/postprocessing.py]: Add a blank new line at the end of the file.

File: index.php (hunk 2 above)
Comment (franciswebb1) [reply index.php]: Have updated

**Pull request merged**